### PR TITLE
Add more resources to the D2k mod

### PR DESCRIPTION
**Pull request by penev92**
1. With the map importer for Dune 2000 maps in the making (#7135), we will need a few more things:
   **a) add "missing" tiles to the `arrakis.yaml` tileset file**
   While we (_may, but questionably_) have all different tiles referenced in the file and this is good enough for the map editor, the map import does not have everything it needs (terrain-wise), which results in incorrect tiles (nothing deal-breaking, but they have to be then manually replaced via the map editor). I've gone over this in several other tickets and on IRC. (#5875)
   **b) add House Corrino, Fremen, Smugglers and Mercenaries as non-playable subfactions**
   This is required for importing missions.
2. Also I am adding a few voice notifications that we are not, but should be using.
   a) The code that will use the `EnemyUnitsApproaching` notification is currently being worked on.
   b) `Guarding` should really be used by the existing Guard command.
   c) Once the new UI is in place and the upgrades for the production structures are implemented, the `Upgrading` notification should be used. (#6043, #3211)
3. Please don't bash me for the empty `corrino.yaml`. I will need some advice on what structures does this faction have that are visually different from the other houses to populate it properly.
4. Added a few more loading screen messages :)

### CODE REVIEW

**Comment by penev92** [comment]
Removed the controversial tileset template additions in order to get this merged.
(The tileset issue will be resolved once I have time to get back to the map improter.)

File: mods/d2k/tilesets/arrakis.yaml [hunk 1]
```diff
@@ -6364,4 +6364,5 @@ Templates:
 		PickAny: False
 		Tiles:
 			0: Rock
-
+			
+
```
Comment (Mailaender) [inline mods/d2k/tilesets/arrakis.yaml]: spurious newline changes

**Comment by penev92** [comment]
Removed newlines, squashed the commits.

File: mods/d2k/tilesets/arrakis.yaml [hunk 2]
```diff
@@ -6363,5 +6363,4 @@ Templates:
 		Category: Sand-Platform
 		PickAny: False
 		Tiles:
-			0: Rock
-
+			0: Rock
```
Comment (Mailaender) [inline mods/d2k/tilesets/arrakis.yaml]: Junk change please revert.

**Comment by penev92** [comment]
Reverted the tileset file entirely.

**Comment by Mailaender** [comment]
:+1:

**Comment by Mailaender** [comment]
Added the `Guarding` voice myself in https://github.com/OpenRA/OpenRA/pull/7257.

**Comment by obrakmann** [comment]
:+1:

**Pull request merged**